src/gui/addnewtorrentdialog.cpp

```diff
@@ -961,6 +961,21 @@ void AddNewTorrentDialog::setupTreeview()
             currentIndex = m_contentModel->index(0, 0, currentIndex);
             m_ui->contentTreeView->setExpanded(currentIndex, true);
         }
+
+        // Check file name blacklist for torrents that are manually added
+        QVector<BitTorrent::DownloadPriority> priorities = m_contentModel->model()->getFilePriorities();
+        Q_ASSERT(priorities.size() == m_torrentInfo.filesCount());
+
+        for (int i = 0; i < priorities.size(); ++i)
+        {
+            if (priorities[i] <= BitTorrent::DownloadPriority::Ignored)
+                continue;
+
```

Comment:
Strange to see `<` here.
Perhaps it isn't really needed?
```suggestion
```
The same to other similar code instances.